id/build.gradle.kts

```diff
@@ -36,11 +36,14 @@ dependencies {
     implementation(project(":feature:orchestrator"))
     implementation(project(":feature:dashboard"))
 
+    implementation(project(":fingerprint:infra:scanner"))
```

Comment:
@BurningAXE , are there any other options instead of adding a scanner dependancy in the ID module?

Reply:
Yeah, didn't like it either. However, the firmware worker is (for obvious reasons) inside the scanner module. If we want to avoid this dependency we can move the scheduling to another place (maybe the Orchestrator) and do it on every workflow. However, I don't like this solution for two reasons:
1. It will scatter worker scheduling in different places, making it harder to reason about workers, etc
2. Similar to why we moved the worker scheduling to app startup - if the user doesn't trigger any workflows for a while it will lead to the firmware worker incorrectly (not) triggering.

So, if there isn't any hard requirement not to make `id` depend on the scanner module, I think this is the lesser of the two evils. Feel free to suggest anything not evil, though :D